pkgs/applications/networking/mailreaders/thunderbird/packages.nix

```diff
@@ -51,6 +45,30 @@ rec {
       })];
     });
   };
```

Comment:
https://bugzilla.mozilla.org/show_bug.cgi?id=1790071 is closed two years ago; this doesn't seem relevant for 128?

Reply:
By my reading they just patched their vendored copy of ICU.  We could choose to use the vendored one.  The upstream ticket seems unresolved: https://unicode-org.atlassian.net/browse/ICU-22132